WebAPI/Startup.cs

```diff
@@ -1,34 +1,35 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+using Application.Dependency_Injection;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.HttpsPolicy;
-using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
+using Persistence.Contexts;
+using Persistence.Repositories;
+using Persistence.Repository_Interfaces;
 
 namespace WebAPI
 {
     public class Startup
     {
+        private IConfiguration Configuration { get; }
+        
         public Startup(IConfiguration configuration)
         {
             Configuration = configuration;
         }
 
-        public IConfiguration Configuration { get; }
-
-        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddApplication();
             services.AddControllers();
+            services.AddScoped<IUserRepository, UserRepository>();
+            services.AddDbContext<BlogContext>(options => options
+                .UseSqlServer(Configuration.GetConnectionString("BlogConnection"))
+                .UseLazyLoadingProxies());
```

Comment:
Which reason you have to use LazyLoadings?

Reply:
I have confused about some things. There is really no need =)